### PR TITLE
Workload Identity: Add protos for issuance service and Attributes

### PR DESCRIPTION
**Pull request by strideynet**
Part of https://github.com/gravitational/teleport/issues/49574

As per [RFD 191](https://github.com/gravitational/teleport/blob/master/rfd/0191-workload-id-config-ux.md)

Broke these protos out from a WIP PR since it was already getting pretty large.

### CODE REVIEW

**Review by tigrato — APPROVED** [review]

File: api/proto/teleport/workloadidentity/v1/issuance_service.proto [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+syntax = "proto3";
+
+package teleport.workloadidentity.v1;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "teleport/workloadidentity/v1/attrs.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1pb";
```
Comment (espadolini) [inline api/proto/teleport/workloadidentity/v1/issuance_service.proto:23]: We've been following the buf managed mode convention for new proto packages, which should make this go package name `workloadidentityv1`, not `workloadidentityv1pb`.

```suggestion
option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1";
```

File: api/proto/teleport/workloadidentity/v1/attrs.proto [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.workloadidentity.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1pb";
```
Comment (espadolini) [inline api/proto/teleport/workloadidentity/v1/attrs.proto:19]:
```suggestion
option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1";
```

Comment (strideynet) [reply api/proto/teleport/workloadidentity/v1/attrs.proto]: [57cb7aa](https://github.com/gravitational/teleport/pull/50163/commits/57cb7aaed8e5339ea968c544cef7e3853b129c60)

File: api/proto/teleport/workloadidentity/v1/issuance_service.proto (hunk 1 above)
Comment (strideynet) [reply api/proto/teleport/workloadidentity/v1/issuance_service.proto]: [57cb7aa](https://github.com/gravitational/teleport/pull/50163/commits/57cb7aaed8e5339ea968c544cef7e3853b129c60)

**Review by espadolini — APPROVED** [review]

**Pull request merged**